src/Compositor.cpp

```diff
@@ -2044,7 +2053,7 @@ void CCompositor::moveWorkspaceToMonitor(CWorkspace* pWorkspace, CMonitor* pMoni
 
             Debug::log(LOG, "moveWorkspaceToMonitor: Plugging gap with new {}", nextWorkspaceOnMonitorID);
 
-            g_pCompositor->createNewWorkspace(nextWorkspaceOnMonitorID, POLDMON->ID);
+            g_pCompositor->createNewWorkspace(nextWorkspaceOnMonitorID, POLDMON->ID, "");
```

Comment:
?

Reply:
Leftover from the removal of 0bbad21d7b64bed58674ce1cfbbdbe6369ab6091, my bad. Addressed in d8d258ea3d152cf831f152feaa668ff8f001fad9